### ISSUE TITLE
Ublock origin shows "?" in some fields, options not working, adds seem blocked

### ISSUE THREAD

**Issue by zeridon** [gorhill/uBlock#1204]
Hi,

I am using chromium 37.0.2062.120-0ubuntu0.12.04.1~pkg917 on ubuntu 12.04 and for some time (around a week i think) tuning settings for ublock origin has been impossible.

The badge displays some numbers but no settings can be changed, network log is unavailable, whitelists, custom block rules and so are missing. No buttons (besides the tabs in settings are working)

The only pointer i have is: 
```
chrome-extension://cjpalhdlnbpafiamejdnhcphjbkeiagm/js/vapi-client.js:35
```

This opens a js file that looks from ublock origin.

Adds/trackers themselves seem blocked. (at least i see the blocks in developer console).

I've tried:
 * Dis/Enable extension
 * Un/Install extension
 * browser restart

No idea when was the last time the browser/extension was updated

Some screencaps attached
![ss-1](https://cloud.githubusercontent.com/assets/5081863/12223647/8b51ffe4-b7e5-11e5-9995-f0e2d08ffafe.png)
![ss-2](https://cloud.githubusercontent.com/assets/5081863/12223650/8b56f288-b7e5-11e5-8209-dcef4af3cde4.png)
![ss-3](https://cloud.githubusercontent.com/assets/5081863/12223648/8b53da26-b7e5-11e5-91d3-b1631c286bba.png)
![ss-4](https://cloud.githubusercontent.com/assets/5081863/12223649/8b54506e-b7e5-11e5-8082-40a1c78e2fd8.png)

**Comment by zeridon** [comment]
seems to be a duplicate of #1124 

Not sure why extension is not updated. Feel free to close if extension has been updated in the store

**Comment by gorhill** [comment]
Fixed in 19518aed607718fc6106e1cf34e59cf18af6bedf, published to Chrome store in 1.5.6.

**Comment by zeridon** [comment]
Thanks for the fast reaction :) works fine :+1:
